src/App.js

```diff
@@ -22,6 +23,7 @@ const App = () => (
       <Route path="/signup" element={<SignUpForm />} />
       <Route path="/reservations" element={<Reservations />} />
       <Route path="/reservations/new" element={<NewReservationForm />} />
+      <Route path="/navigation" element={<Navigation />} />
     </Routes>
   </Provider>
```

Comment:
- To add the navigation panel in all pages I belive this routes configuration should look like this 🔽 

```
<Provider store={store}>
    <Navigation />
    <Routes>
      <Route path="/" element={<Main />} />
      <Route path="/signin" element={<Login />} />
      <Route path="/signup" element={<SignUpForm />} />
      <Route path="/reservations" element={<Reservations />} />
      <Route path="/reservations/new" element={<NewReservationForm />} />
    </Routes>
  </Provider>
```

after that all pages styles should be adjusted to match the design properly.